### PR TITLE
Add xiaomi GZCGQ01LM illuminance sensor

### LINKED ISSUE #1306: Zigbee2mqtt: Add device GZCGQ01LM
```
{
  "name": "0x54ef4410000c1618",
  "external_id": "zigbee2mqtt:0x54ef4410000c1618",
  "model": "GZCGQ01LM",
  "features": [
    {
      "category": "battery",
      "type": "integer",
      "unit": "percent",
      "name": "Battery",
      "read_only": true,
      "has_feedback": false,
      "min": 0,
      "max": 100,
      "zigbeeField": "battery",
      "external_id": "zigbee2mqtt:0x54ef4410000c1618:battery:integer:battery",
      "selector": "zigbee2mqtt:0x54ef4410000c1618:battery:integer:battery"
    }
  ],
  "should_poll": false,
  "service_id": "50ad3c37-ce5a-4c49-9ec7-0ed195e1f0c6",
  "supported": false
}
```

### PR DESCRIPTION
**Pull request by vincentBesseau**
fix #1306

@cicoub13, @atrovato, @Pierre-Gilles

### CODE REVIEW

File: server/services/zigbee2mqtt/utils/features.js [hunk 1]
```diff
@@ -77,6 +77,17 @@ const features = {
...
   },
+  illuminance_lux: {
+    category: DEVICE_FEATURE_CATEGORIES.LIGHT_SENSOR,
+    type: DEVICE_FEATURE_TYPES.SENSOR.DECIMAL,
+    unit: DEVICE_FEATURE_UNITS.LUX,
+    name: 'Light Sensor',
+    read_only: true,
+    has_feedback: false,
+    min: 0,
+    max: 1000,
```
Comment (Pierre-Gilles) [inline server/services/zigbee2mqtt/utils/features.js]: Is it really from 0 to 1000 ? 

https://fr.wikipedia.org/wiki/Lux_(unit%C3%A9)

**Review by Pierre-Gilles — COMMENTED** [review]
Just a little comment, otherwise I let the Zigbee expert review this PR, I know nothing about the integration :)

File: server/services/zigbee2mqtt/utils/features.js (hunk 1 above)
Comment (vincentBesseau) [reply server/services/zigbee2mqtt/utils/features.js]: Hi 👋
As I can see in your link, it's not enough 
Screenshot_20211018-073912
Would you like to rise the max value ?

**Review by VonOx — APPROVED** [review]
Can be merged

File: server/services/zigbee2mqtt/utils/features.js (hunk 1 above)
Comment (Pierre-Gilles) [reply server/services/zigbee2mqtt/utils/features.js]: Yes

**Comment by Pierre-Gilles** [comment]
As said in private, I'm ok with the PR but let's synchronize with @atrovato PR, this PR might be unnecessary

**Comment by Pierre-Gilles** [comment]
FYI, as I'm doing a release soon, I merged this so at least it's in the release :)

**Pull request merged**